packages/grafana-ui/src/components/Forms/Field.tsx

```diff
@@ -12,7 +12,7 @@ export interface FieldProps extends HTMLAttributes<HTMLDivElement> {
   /** Label for the field */
   label?: React.ReactNode;
   /** Description of the field */
-  description?: string;
+  description?: React.ReactNode;
```

Comment:
I'm not sure we should apply this change. See my comment where we use the field that requires this change.

Reply:
I think this is ok, A coming enhancement in the panel edit search function is to highlight character hits in field label & description and for that we need React.ReactNode here

Reply:
ok ok, super!